openmls/src/group/mls_group/builder.rs

```diff
@@ -194,22 +195,6 @@ impl MlsGroupBuilder {
         self
     }
 
-    /// Sets the `external_senders` property of the MlsGroup.
-    pub fn external_senders(mut self, external_senders: ExternalSendersExtension) -> Self {
```

Comment:
Remmoving these public functions should go into the changelog.
We should add the others we added as well.